gadopt/gplates/gplates.py

```diff
@@ -404,7 +408,7 @@ def _calc_velocities(self, velocity_domain_features, topology_features, rotation
                         # add it to the list
                         all_velocities.extend(velocity_vectors)
                     else:
-                        warnings.warn("Issues finding plate ids for some of the seeds.", category=RuntimeWarning)
+                        warnings.warn("pygplates couldn't assign plate IDs to some seeds due to irregularities in the reconstruction model. G-ADOPT will interpolate the nearest values.", category=RuntimeWarning)
```

Comment:
This is an attempt to address the issue raised in https://github.com/g-adopt/g-adopt/issues/105.

The issue is solely sue to some issues in the specific plate reconstruction. There is nothing we could do about it at this stage, but at the same time it is important to give a warning about this as we then need to interpolate for those unknown points. 

I have now tried to be more clear about the issue in the warning message hope that helps.